02 - Community Expansions/02.05 All Community Expansions/Auxiliary Tools/DSS_Folder_Maker.md

```diff
@@ -0,0 +1,21 @@
+---
+aliases:DSS_Folder_Maker
```

Comment:
For my own education, what's the benefit of giving an alias that is (as far as I can see) the same as the file name?